pennylane_lightning/core/src/bindings/Bindings.cpp

```diff
@@ -85,6 +85,6 @@ PYBIND11_MODULE(
     // Register bindings for backend-specific info:
     registerBackendSpecificInfo(m);
 
-    registerLightningTensorClassBindings<TensorNetBackends>(m);
+    registerLightningTensorClassBindings<AllTensorNetBackends>(m);
```

Comment:
I don't think we need "All" as TensorNetwork can be applied to any tensor-network backend
```suggestion
    registerLightningTensorClassBindings<TensorNetworkBackends>(m);
```

Reply:
Done!